ext/oci8/oci8_interface.c

```diff
@@ -757,9 +749,10 @@ PHP_FUNCTION(oci_lob_copy)
 {
 	zval *tmp_dest, *tmp_from, *z_descriptor_dest, *z_descriptor_from;
 	php_oci_descriptor *descriptor_dest, *descriptor_from;
-	zend_long length = 0;
+	zend_long length = -1;
```

Comment:
Don't we have the same issue with the default value here?

Reply:
Yes, I didn't notice it

Reply:
I fixed the same problem in `oci_lob_export()` as well.